COLLABORATOR_GUIDE.md

```diff
@@ -263,6 +263,24 @@ multiple commits. Commit metadata and the reason for the revert should be
 appended. Commit message rules about line length and subsystem can be ignored.
 A Pull Request should be raised and approved like any other change.
 
+### Introducing New Modules
+
+Semver-minor commits that introduce new core modules should be treated with
+extra care.
+
+The name of the new core module should not conflict with any existing 
+module in the ecosystem unless a written agreement with the owner of those
+modules is reached to transfer ownership.
+
+New core modules must be landed with a [Stability Index][] of Experimental.
+Pull requests proposing new core modules must still be marked semver-minor.
+New core modules must remain Experimental until a semver-major release.
+
+Pull requests introducing new core modules must be left open for at least
+one week for review. For new modules that involve significant effort,
+non-trivial additions to Node.js or significant new capabilities, an
+[Enhancement Proposal][] is recommended but not required.
```

Comment:
This sentence is confusing to me. (For example, it implies that new modules typically are trivial additions to Node.js.)

How about this?:

> If a new module has a large API surface, requires extensive changes to Node.js core, or adds extraordinary new functionality, open an [Enhancement Proposal][].